src/legacy/core_plugins/metrics/public/components/index_pattern.js

```diff
@@ -100,45 +131,38 @@ export const IndexPattern = props => {
               onChange={handleSelectChange(timeFieldName)}
               indexPattern={model[indexPatternName]}
               fields={fields}
-              placeholder={isDefaultIndexPatternUsed ? model.default_timefield : undefined}
```

Comment:
@Avinar-24  Are you sure you want to remove this placeholder?

Reply:
Reverted